### DISCUSSION TITLE
[RAC] Select with modal overlay doesn't work

### DISCUSSION THREAD

**Discussion by mehdibha** [adobe/react-spectrum#6409] [Q&A]
Hi there,
I am seeing that Select can not be used with Modal overlay but only with popover, is there a way to do this and still using the RAC components but maybe with a custom modal using hooks?
This is also the case of the combobox where i'll sureley follow how it was built in react spectrum.
I didn't understand how was the popover triggered but not the modal as they both use `OverlayTriggerStateContext` 😒

**Comment by snowystinger** [comment]
closed as discussion is in https://github.com/adobe/react-spectrum/issues/6021
